09-epidemiologic.Rmd

```diff
@@ -1,20 +1,428 @@
-#  Epidemiologic studies on the heat effects     {#he2}
+---
```

Comment:
Header und einführung bitte wie hier:
https://github.com/henrifnk/Seminar_ClimateNStatistics/blob/master/07-hydroLSTM.Rmd
Dann Caption so, wie im Bericht vorgesehen:
# Epidemiologic studies on the heat effects {#he2}
Un das noch dazu:
*Degree: Bachelor*